<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DifficultyViewPage</class>
 <widget class="QWidget" name="DifficultyViewPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="NoOutSideViews">
     <property name="text">
      <string>No outside views</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="NoSelfView">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>No self view</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="NoFoeView">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>No foe view</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="NoFriendlyView">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>No friendly view</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="NoPlanesView">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>No planes view</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="NoACarrierView">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>No aircraft carrier view</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="CockpitAlwaysOn">
     <property name="text">
      <string>Cockpit always on</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="NoSpeedBar">
     <property name="text">
      <string>No speed bar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="NoPadlock">
     <property name="text">
      <string>No padlock</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
